models/intermediate/int_apple_search_ads__most_recent_ad_group.sql

```diff
@@ -4,11 +4,11 @@ with base as (
 ), filtered as (
     select * 
     from base
-    {% if target.type == 'snowflake' -%}
-        where is_most_recent_record = 'true'
-    {% else -%}
-        where is_most_recent_record is true
-    {% endif %}
+    {% if target.type == 'snowflake' -%} --
+        where is_most_recent_record = 'true' --
+    {% else -%} --
+        where is_most_recent_record is true --
+    {% endif %} --
```

Comment:
Curious why we need this conditional?

Could we not instead use:
```sql
where is_most_recent_record
```
to get around the need for the Snowflake conditional?

Reply:
Great call out, Joe! I meant to test this but slipped my mind.